apps/judicial-system/web/src/components/Header/Header.tsx

```diff
@@ -1,63 +1,118 @@
   const { isAuthenticated, user } = useContext(UserContext)
-  const { width } = useWindowSize()
-  const isMobile = width <= theme.breakpoints.md
+  const logoHref =
+    !user || !isAuthenticated
+      ? '/'
+      : user.role === UserRole.DEFENDER
+      ? `${constants.DEFENDER_ROUTE}/${router.query.id}`
+      : user.role === UserRole.ADMIN
+      ? constants.USER_LIST_ROUTE
+      : constants.CASE_LIST_ROUTE
+
+  const handleLogout = async () => {
+    await api.logout()
+    window.location.assign('/')
+  }
 
   return (
-    <header className={styles.header}>
-      <Link
-        href={
-          !user || !isAuthenticated
-            ? '/'
-            : user.role === UserRole.DEFENDER
-            ? `${constants.DEFENDER_ROUTE}/${router.query.id}`
-            : user.role === UserRole.ADMIN
-            ? constants.USER_LIST_ROUTE
-            : constants.CASE_LIST_ROUTE
-        }
-        data-testid="link-to-home"
-      >
-        <Box display="flex" cursor="pointer" className={styles.logoContainer}>
-          <Logo width={isMobile ? undefined : 146} iconOnly={isMobile} />
-          {router.pathname !== '/' && (
-            <span className={styles.logoContainerRvgName}>
-              <Text>Réttarvörslugátt</Text>
-            </span>
-          )}
-        </Box>
-      </Link>
-      {isAuthenticated && (
-        <Button
-          variant="ghost"
-          icon="logOut"
-          iconType="outline"
-          size="small"
-          onClick={async () => {
-            await api.logout()
-            window.location.assign('/')
-          }}
-          data-testid="logout-button"
-        >
-          {user?.name}
-        </Button>
-      )}
-    </header>
+    <Box paddingX={[0, 0, 4]}>
+      <GridContainer className={styles.container}>
+        <GridRow>
+          <GridColumn span="12/12">
+            <Header
+              info={{
+                title: 'Dómsmálaráðuneytið',
+                description: 'Réttarvörslugátt',
+              }}
+              logoRender={(logo) => (
+                <Link href={logoHref}>
+                  <a href={logoHref}>{logo}</a>
+                </Link>
+              )}
+              headerItems={
+                <UserMenu
+                  language="is"
+                  authenticated={isAuthenticated}
+                  username={user?.name}
+                  dropdownItems={
+                    <>
+                      <div className={styles.dropdownItem}>
+                        <Box marginRight={2}>
+                          <Icon icon="person" type="outline" color="blue400" />
+                        </Box>
+                        <Box>
+                          <Box marginBottom={2}>
+                            <Text>{capitalize(user?.title || '')}</Text>
+                          </Box>
+                          <Box marginBottom={2}>
+                            <Text>{user?.institution?.name}</Text>
+                          </Box>
```

Comment:
User info is always provided for regular users, but not for defenders. I think we need add something like `?? ''` to avoid `undefined` to be displayed.